src/utils/utils.tsx

```diff
@@ -115,6 +115,7 @@ export const ModelFieldsTable: any = {
   memory_gb: "Memory (GB)",
   storage: "Storage",
   comment: "Comment",
+  model_type: "Mount Type"
```

Comment:
similarly, i think this should be called Mount Type, not model type, but keeping variable names consistent with backend for now

Reply:
You're tracking changing this assuming Bletsch confirms?

Reply:
yeet